cmd/cp-main.go

```diff
@@ -338,9 +338,19 @@ func doCopySession(session *sessionV8) error {
 					gracefulStop()
 					return
 				}
+
+				if e := urlScanner.Err(); e != nil {
+					// Error while reading. quit immediately
+					gracefulStop()
+					return
+				}
+
 				var cpURLs URLs
 				// Unmarshal copyURLs from each line.
-				json.Unmarshal([]byte(urlScanner.Text()), &cpURLs)
+				if e := json.Unmarshal([]byte(urlScanner.Text()), &cpURLs); e != nil {
+					errorIf(probe.NewError(e), "Unable to unmarshal %s", urlScanner.Text())
+					continue
```

Comment:
It is not clear if this is an odd situation or can be easily triggered by a user input, can't we just exit here ?

Reply:
We can exit, but my idea to throw an error and continue if possible for other entries which might be valid @vadmeste - so will throw an error and proceed forward.